contrib/inventory/gce_googleapiclient.py

```diff
@@ -217,6 +217,17 @@ def signal_handler():  # pragma: no cover
     Random.atfork()
 
 
+def check_project(project_billing_info):
```

Comment:
i think we can have a more descriptive name on this function (what do you think about `is_project_active()`?)

Reply:
Sure, I'll do it now.